### PR TITLE
updated latest dashboard image

### PR DESCRIPTION
**Pull request by ankitsri11**

### CODE REVIEW

File: navigation/pipelines_dashboard_page.md [hunk 1]
```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
...
-1.  The pipelines are listed under the pipeline group that they belong to
-2.  Click the pipeline name to see [pipeline activity](../navigation/pipeline_activity_page.md) for that pipeline.
-3.  Click pipeline instance label to see details for that instance.
-4.  Click on each stage segment to see [stage details](../navigation/stage_details_page.md) for that stage instance.
-5.  "triggered by [user name] about [how long ago]" gives you a quick look at who activated this pipeline and when this was triggered.
-6.  The "Trigger" button forces a pipeline to begin build activity
-7.  The [Trigger with Options](../advanced_usage/trigger_with_options.md) button allows the you to pick the revisions of materials that the pipelines should build with, and trigger the pipeline.
-8.  The "Pause" button pauses scheduling of the pipeline.
+1.  The pipelines are listed under the pipeline group that they belong to. You can also edit pipeline group by clicking on settings icon.
+2.  Pipeline instance [label](../configuration/build_labelling.md) which can be equivalent to pipeline run.
```
Comment (arvindsv) [inline navigation/pipelines_dashboard_page.md]: "Pipeline run" is not defined anywhere. Maybe just say: "... defaults to the number of times a pipeline has run" or something similar?

File: navigation/pipelines_dashboard_page.md [hunk 2]
```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
...
-5.  "triggered by [user name] about [how long ago]" gives you a quick look at who activated this pipeline and when this was triggered.
-6.  The "Trigger" button forces a pipeline to begin build activity
-7.  The [Trigger with Options](../advanced_usage/trigger_with_options.md) button allows the you to pick the revisions of materials that the pipelines should build with, and trigger the pipeline.
-8.  The "Pause" button pauses scheduling of the pipeline.
+1.  The pipelines are listed under the pipeline group that they belong to. You can also edit pipeline group by clicking on settings icon.
+2.  Pipeline instance [label](../configuration/build_labelling.md) which can be equivalent to pipeline run. 
+3.  Click on each stage segment to see [stage details](../navigation/stage_details_page.md) for that stage instance.
+4.  The "Trigger" button forces a pipeline to begin build activity.
+5.  The [Trigger with Options](../advanced_usage/trigger_with_options.md) button allows the you to pick the revisions of materials that the pipelines should build with, and trigger the pipeline.
```
Comment (arvindsv) [inline navigation/pipelines_dashboard_page.md]: Typo: "allows the you".

File: navigation/pipelines_dashboard_page.md [hunk 3]
```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
...
+6.  The "Pause" button pauses scheduling of the pipeline.
+7.  If you are a GoCD pipeline group administrator or a super administrator, you can now navigate to edit a pipeline by clicking this settings icon on the pipeline dashboard or the environments page.
```
Comment (arvindsv) [inline navigation/pipelines_dashboard_page.md]: I'd remove the part about the environments page. That pipeline view is actually scheduled to go away soon.

File: navigation/pipelines_dashboard_page.md [hunk 4]
```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
...
+8.  View all changes between the current pipeline instance with the previous one.
```
Comment (arvindsv) [inline navigation/pipelines_dashboard_page.md]: "View all changes between the current pipeline instance and any previous one."?

**Comment by ankitsri11** [comment]
@arvindsv - Made the changes as requested. Can you please review and let me know if any other changes required. Thanks.

**Comment by ketan** [comment]
Can you not checkin the yarn.lock files (squash the commits please).

**Comment by ankitsri11** [comment]
@ketan - I did here - https://github.com/gocd/docs.go.cd/pull/183/commits/c560063d0cc5015fed2c71726e391797bf47165c. Do I still need to squash?

**Comment by ketan** [comment]
You removed the actual [yarn.lock](https://github.com/gocd/docs.go.cd/pull/183/files#diff-8ee2343978836a779dc9f8d6b794c3b2) and added a blank one [here](https://github.com/gocd/docs.go.cd/pull/183/files#diff-6c2d8e1609c35008907c986974b1c13b)

File: navigation/pipelines_dashboard_page.md [hunk 5]
```diff
@@ -12,21 +12,24 @@ This shows the current activity in the system. You can see all the instances of
...
-11. This shows you the name and status of the last executed stage in that pipeline instance.
-12. View all changes between the current pipeline instance with the previous one.
-13. Lets you customize which pipelines are displayed on the dashboard. See [below](#personalize-pipelines-view) for more details.
-14. Lets you search for any pipeline configured to be visible on your pipeline dashboard.
-15. If you are a GoCD pipeline group administrator or a super administrator, you can now navigate to edit a pipeline by clicking this settings icon on the pipeline dashboard or the environments page.
+10. Click the pipeline VSM to see the [value stream map](../navigation/value_stream_map.md) of that instance of the pipeline.
+11. Click the pipeline history to see [pipeline activity](../navigation/pipeline_activity_page.md) for that pipeline.
+12. "triggered by [user name] about [how long ago]" gives you a quick look at who activated this pipeline and when this was triggered.
```
Comment (arvindsv) [inline navigation/pipelines_dashboard_page.md]: The "about" part is no longer true.

**Comment by ankitsri11** [comment]
Closing this one as created a new PR with different branch.

**Pull request closed without merging**